data/lang/module-cargorun/en.json

```diff
@@ -213,35 +217,35 @@
   },
   "HOWMUCH_1": {
     "description": "regarding mass, always more than 1t",
-    "message": "{amount} metric tons."
+    "message": "Yes it is. The total amount is {amount} metric tons."
   },
   "HOWMUCH_2": {
     "description": "regarding mass, always more than 1t",
-    "message": "{amount}t"
+    "message": "Yes, we can split the cargo. I have {amount}t in total."
   },
   "HOWMUCH_3": {
     "description": "regarding mass, always more than 1t",
-    "message": "The mass is {amount}t."
+    "message": "Yes. The entire mass is {amount}t."
   },
   "HOWMUCH_4": {
     "description": "regarding mass, always more than 1t",
-    "message": "There are {amount}t."
+    "message": "There are {amount}t. We can split the cargo run if you want."
   },
   "HOWMUCH_SINGULAR_1": {
     "description": "This string is only used for 1 metric tons (singular)",
-    "message": "It's only {amount}t."
+    "message": "No it is not. It's only {amount}t."
   },
   "HOWMUCH_WHOLESALER_1": {
     "description": "",
-    "message": "There are {amount}t of {cargoname}."
+    "message": "There are {amount}t of {cargoname}. I can offer the following freight runs:"
   },
   "HOWMUCH_WHOLESALER_2": {
     "description": "",
-    "message": "{amount}t {cargoname}"
+    "message": "{amount}t {cargoname} can be divided as follows:"
   },
   "HOW_MUCH_MASS": {
     "description": "",
-    "message": "How much mass?"
+    "message": "Is the amount of cargo negotiable?"
```

Comment:
I haven't playtested, and maybe this is needed for the code to enter a menu system, but if not, one could simply have the player, instead of asking to negotiate, make a counter-deal right away: 
"Unfortunately I can only take:
- [x] 10t
- [ ] 100t 
- [ ] 1000t"

Reply:
Yes, it is needed to enter the menu with the amounts.